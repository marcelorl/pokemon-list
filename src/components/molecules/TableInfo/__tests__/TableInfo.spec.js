import createTestContext from 'react-cosmos-test/enzyme'

import fixture from '../TableInfo.fixture'

const { mount, getWrapper } = createTestContext({ fixture })

beforeEach(mount)

describe('#TableInfo', () => {
  it('renders component', () => {
    expect(getWrapper()).toMatchSnapshot()
  })

  it('should have all data', () => {
    expect(getWrapper().find('thead th').text()).toEqual('Title')
    expect(getWrapper().find('tbody tr')).toHaveLength(3)
  })
})
