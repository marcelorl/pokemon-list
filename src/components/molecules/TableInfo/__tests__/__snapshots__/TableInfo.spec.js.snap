// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`#TableInfo renders component 1`] = `
<TableInfo
  items={
    Array [
      1,
      2,
      3,
    ]
  }
  title="Title"
>
  <Table
    responsiveTag="div"
    tag="table"
  >
    <table
      className="table"
    >
      <thead>
        <tr>
          <th>
            Title
          </th>
        </tr>
      </thead>
      <tbody>
        <tr
          key="0"
        >
          <td>
            1
          </td>
        </tr>
        <tr
          key="1"
        >
          <td>
            2
          </td>
        </tr>
        <tr
          key="2"
        >
          <td>
            3
          </td>
        </tr>
      </tbody>
    </table>
  </Table>
</TableInfo>
`;
